django-stubs/views/generic/list.pyi

```diff
@@ -17,7 +17,7 @@ class MultipleObjectMixin(ContextMixin):
     paginator_class: Type[Paginator] = ...
     page_kwarg: str = ...
     ordering: Sequence[str] = ...
-    def get_queryset(self) -> QuerySet: ...
+    def get_queryset(self) -> Union[Iterable, QuerySet]: ...
```

Comment:
`Iterable` is generic. Can you please add an argument there?
And also a test case for it. Because queryset part is highly complex.